### PR TITLE
DOCSP-20589-release-notes-for-param-update

### PR DESCRIPTION
**Pull request by davemungo**
JIRA
https://jira.mongodb.org/browse/DOCSP-20589
Release note to accompany DOCSP-18668

STAGING
https://docs-mongodbcom-staging.corp.mongodb.com/mongodb-shell/docsworker-xlarge/DOCSP-20589-release-notes-for-update/changelog/#v1.0.6

### CODE REVIEW

File: source/changelog.txt [hunk 1]
```diff
@@ -181,6 +181,10 @@ New features in this release:
   ``// comments`` are now preserved as ``/* comments */`` in the
   history entry.
 
+- The :ref:`aggregation-pipeline` is now an optional parameter for
```
Comment (jmd-mongo) [inline source/changelog.txt:184]: [suggestion]

The way this currently renders makes it a bit tricky to parse. 

Spitballing:

"The :ref:`pipeline parameter <aggregation-pipeline>` paramater is now optional for :method:`db.collection.watch()`, :method:`db.watch()`, and :method:`Mongo.watch()`."

**Review by jmd-mongo — COMMENTED** [review]
Thanks for the changes, @davemungo! I've left a suggestion for you.

Thanks,
Joe

File: source/changelog.txt (hunk 1 above)
Comment (davemungo) [reply source/changelog.txt:184]: Done

**Review by davemungo — COMMENTED** [review]
Thanks for the suggestion @jmd-mongo 

This is ready for another pass when you have a moment.

**Review by jmd-mongo — APPROVED** [review]
LGTM!

Thanks,
Joe

**Review by addaleax — APPROVED** [review]

**Pull request merged**